constants/members.json

```diff
@@ -3152,29 +3160,29 @@
     },
     {
       "name": "Ayush Shah",
-      "role": "vp talent",
-      "term": 1235,
-      "teams": ["executive"],
-      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fs23headshots%2FVPT_AyushShah.png?alt=media&token=52921c5a-fd94-4daa-93d3-5dae93f1756f&_gl=1*bleofo*_ga*MTUwMDEyNjY3My4xNjc0MDgyODE2*_ga_CW55HF8NVT*MTY4NTkwMDY2Mi4zLjEuMTY4NTkwMTgzNC4wLjAuMA.."
+      "role": "developer",
+      "term": 1236,
+      "teams": ["abtc"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVPT_AyushShah.png?alt=media&token=d67e7fde-b7fb-42d8-b9af-ac3f254ff75e"
     },
     {
       "name": "Tracy Wai",
-      "role": "vp talent",
-      "term": 1235,
+      "role": "vp scoping",
+      "term": 1236,
       "teams": ["executive"],
       "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fs23headshots%2FVPT_Tracy_Wai.jpeg?alt=media&token=f8bdd126-24b6-4594-80ca-7f4440933725&_gl=1*4frx90*_ga*MTUwMDEyNjY3My4xNjc0MDgyODE2*_ga_CW55HF8NVT*MTY4NTkwMDY2Mi4zLjEuMTY4NTkwMTg1OC4wLjAuMA.."
     },
     {
       "name": "Edward Ho",
       "role": "vp scoping",
```

Comment:
edward (eddie) ho is actually pm for abtc now, so youll need to update this

Reply:
main spreadsheet is wrong? @heet